pages/phases/getting-started/collecting-data.md

```diff
@@ -15,10 +15,13 @@ sub-phase-order: 040
   {{ page.intro }}
 </p>
 
-For one to two recent DE actions for a similar GS level and series to the position you plan to fill, gather:
+To gauge the success of your hiring action, collect baseline data about recent hiring actions at your agency. If you plan a delegated examination (DE) pilot, you will want to collect data from past DE actions. You'll compare this information with the results you receive when your new hiring action is complete. 
 
-1. **Current qualified rate**: How many applications were considered qualified by an HR specialist.
-2. **Current selection rate**: The number of selections made off of the resulting certificates, represented numerically and as a percentage of qualified applications.
+For all your DE actions within the past one to two years for a similar GS level and series to the position you plan to fill, gather:
+
+1. **Number of applicants found qualified**: How many applications were considered qualified by an HR specialist.
```

Comment:
dont we want the number vs the rate; and then AFTER they can convert that into a % of selections/qualified (which is within point 3 below)

Reply:
good with me